### ISSUE TITLE
How can ProtoSchool best meet the needs of the Multiformats team? 

### ISSUE THREAD

**Issue by terichadbourne** [multiformats/multiformats#46]
As we build the roadmap for [ProtoSchool](https://proto.school), we'd like to take into account the priorities of the Multiformats team and plan for some tutorial content that best highlights your most common or most prioritized use cases or features. 

For a sense of what ProtoSchool is capable of, please take a look at the [existing tutorials](https://proto.school/#/tutorials), which run in-browser and (with the exception of the first tutorial on the list) offer coding challenges following the introduction of various content step-by-step. Beginner-friendliness is a major priority for this project, and as we consider requests from project teams we will also focus on ensuring that appropriate scaffolding exists to get users to the point where they can successfully approach those proposed topics.

Could you please take a look at your project roadmap and help me understand what ProtoSchool tutorial content might most help you achieve your goals for 2019 and 2020? 

Do you have upcoming events where you hope to offer workshops? If you could envision that content fitting with the ProtoSchool tutorial format, please be sure to include these ideas and share the relevant event dates.

cc @mikeal

**Comment by lgierth** [comment]
Thanks for bringing this up @terichadbourne. Multiformats doesn't currently have any dedicated team or roadmap. We'll get back to the questions at hand at some point :)

What I imagine could be pretty useful is exploratory tooling similar to cid-utils (which I believe is now part of IPLD Explorer?). CIDs are multibase+multicodec+multihash, so cid-utils already covers some good ground. A similar tool could be built for multiaddr. multistream would probably benefit from something more interactive.

Also [The Illustrated TLS 1.3 Connecion](https://tls13.ulfheim.net/) is a very cool concept that could be applied to libp2p connections, and as part of that the various multiformats used there.

**Comment by terichadbourne** [comment]
Thanks so much for weighing in, @lgierth! Just to clarify, when you reference "exploratory tooling" do you mean tools that haven't been built yet, which the Multiformats "team" would create and then ProtoSchool would offer a tutorial about? (Within ProtoSchool our focus is on building educational tutorials around existing technology, as opposed to building and new products or tooling, so just want to make sure we're on the page.) 

cc @mikeal

**Comment by lgierth** [comment]
> which the Multiformats "team" would create and then ProtoSchool would offer a tutorial about

Yes, or maybe some aspects of the tool could be embedded into tutorials.
